src/components/side_panel/chart/building_blocks/title/title.ts

```diff
@@ -47,6 +48,7 @@ interface Props {
   updateAlignment?: (string) => void;
   updateColor?: (Color) => void;
   style: TitleDesign;
+  onFontSizeChanged?: (fontSize: number) => void;
```

Comment:
If this props is optional, there should be a t-if in the template to not display the font size editor if there is no callback. But I dont think there's a reason to make it optional ?